frontend/src/app/modules/landing/landing-routing.module.ts

```diff
@@ -1,12 +1,16 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { RegistrationComponent } from '@modules/landing/registration/registration.component';
 
 import { LandingPageComponent } from './landing-page/landing-page.component';
 
 const routes: Routes = [
     {
         path: '',
         component: LandingPageComponent,
+        children: [
+            { path: 'registration', component: RegistrationComponent },
+        ],
```

Comment:
Can you rename this whole part (with folders) to authorization, we won't have landing?